### DISCUSSION TITLE
Nested bookmarks

### DISCUSSION THREAD

**Discussion by tmdoit** [directus/directus#15521] [Feature Requests]
Hello,
What do you think about nested bookmarks? In my use case it will be helpful to categorize Products collection ie.
```
- Products
  |- Apple
    |- iPhone
        |- iPhone 7
        |- iPhone 8
        ...
    |- iMac
    |- MacBook
    ...
```

**Comment by fitu86** [comment]
I like to do it this way, it gives the feeling of navigating between category pages, it can be used in the Navigation Bar, but also in a singleton with a tree view display.

**Comment by Voldemorten** [comment]
This would be very useful!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
